scripts/buildkite/golint.sh

```diff
@@ -4,15 +4,6 @@ set -ex
 
 make go-generate && make fmt && make lint && make copyright
 
-# buildkite-git complains about "dubious ownership",
-# i.e. user-who-created is different than user-who-statused.
-#
-# that's fine, just override it if it looks like we're in buildkite.
-# elsewhere it's probably best to not touch this, and the path is likely wrong.
-if [ -n "$BUILDKITE" ]; then
-  git config --global --add safe.directory /cadence
```

Comment:
this is now done in the buildkite dockerfile, which appears to catch everything much more simply.